cdap-app-fabric/src/test/java/co/cask/cdap/internal/audit/AuditPublishTest.java

```diff
@@ -124,7 +124,7 @@ public void testPublish() throws Exception {
 
     Multimap<AuditType, EntityId> actualAuditEntities = HashMultimap.create();
     for (AuditMessage message : publishedMessages) {
-      EntityId entityId = message.getEntityId();
+      EntityId entityId = EntityId.fromMetadataEntity(message.getEntity());
```

Comment:
if it is a custom entity, this will return null and the getEntityType() below will throw an NPE.